src/QEDfields.jl

```diff
@@ -13,9 +13,12 @@ export polarization_vector, oscillator
 
 export CosSquarePulse, GaussianPulse
 
+export phase_integral_0, phase_integral_1, phase_integral_2
```

Comment:
We could think about dispatching on a `::Val{N}` type instead of having these three different function names.